packages/css-library/src/utilities/flex-grid.scss

```diff
@@ -0,0 +1,53 @@
+@use 'uswds-core/src/styles/functions' as *;
+@use '../tokens/flex-grid' as *;
+@use './settings' as *;
+
+.vads-grid-container {
```

Comment:
USWDS3 has [flex utilities](https://designsystem.digital.gov/utilities/flex/) and [layout grid utilities](https://designsystem.digital.gov/utilities/layout-grid/), but not a [flex grid like we currently have](https://design.va.gov/foundation/layout/flexbox-grid). If we think that the layout grid utilities from USWDS can be a sufficient replacement, then we probably don't want these.

Reply:
I ended up leaving these in because they're not 1:1 replacements.

Our existing flex grid utility:
```
.vads-flex-col-9 {
  box-sizing: border-box;
  max-width: 75%;
  min-height: 1px;
  min-width: 0;
  position: relative;
  width: 100%;
  flex: 0 0 75%;
}
```

USWDS grid utility:
```
.vads-grid-col-9 {
  flex: 0 1 auto;
  width: 75%;
}
```